bluebrain/repo-bluebrain/packages/spatial-index/package.py

```diff
@@ -29,7 +29,8 @@ class SpatialIndex(PythonPackage):
     version("0.7.0", tag="0.7.0")
     version("0.6.0", tag="0.6.0")
 
-    depends_on("py-setuptools")
+    depends_on("py-scikit-build-core+pyproject", type="build")
+    depends_on("py-setuptools-scm", type="build")
```

Comment:
These should be version-gated like so:
```suggestion
    depends_on("py-setuptools", when="@:2.1.0")
    depends_on("py-scikit-build-core+pyproject", type="build", when="@2.1.1:")
    depends_on("py-setuptools-scm", type="build", when="@2.1.1:")
```